src/app.py

```diff
@@ -98,7 +89,7 @@ def http_internal_server_error(e):
 # The only endpoint that should be in this file, all others should be route Blueprints...
 @app.route('/', methods=['GET'])
 def index():
-    return "Hello! This is HuBMAP File API service :)"
+    return "Hello! This is SenNet File API service :)"
```

Comment:
Since this API is shared by both HuBMAP and SenNet, mentioning either project name may cause confusions. May just omit the project name.

Reply:
Resolved with commit https://github.com/hubmapconsortium/files-api/pull/37/commits/6ac1699ddc88a717662e8315a08cfc98d3abcb3c